sdk/formrecognizer/azure-ai-formrecognizer-perf/src/main/java/com/azure/ai/formrecognizer/perf/ReceiptRecognitionStreamTest.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.perf;
+
+import com.azure.ai.formrecognizer.models.FormRecognizerOperationResult;
+import com.azure.ai.formrecognizer.models.RecognizedForm;
+import com.azure.ai.formrecognizer.perf.core.ServiceTest;
+import com.azure.ai.formrecognizer.perf.core.Utility;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+public class ReceiptRecognitionStreamTest extends ServiceTest<PerfStressOptions> {
+    private FileInputStream fileInputStream;
+    private String receiptFilePath = "src/main/resources/samplesFiles/contoso-receipt.png";
+    private long fileLength;
+    public ReceiptRecognitionStreamTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public Mono<Void> globalSetupAsync() {
+        return super.globalSetupAsync().then(createReceiptInputStream());
+    }
+
+    @Override
+    public Mono<Void> globalCleanupAsync() {
+        return closeReceiptInputStream().then(super.globalCleanupAsync());
+    }
+
+    @Override
+    public void run() {
+        formRecognizerClient.beginRecognizeReceipts(fileInputStream, fileLength).waitForCompletion();
```

Comment:
Watch out on re-using a stream, the read position gets updated every time `read` is called. This may result in the stream reaching its end and all tests after that will fail.